### PR TITLE
IGVF-573-audit-age-sample

### PR DESCRIPTION
**Pull request by khzlin**

### CODE REVIEW

File: src/igvfd/audit/primary_cell.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    audit_checker,
+    AuditFailure,
+)
+from .formatter import (
+    audit_link,
+    path_to_text,
+)
+
+
+@audit_checker('PrimaryCell', frame='object')
```
Comment (gabdank) [inline src/igvfd/audit/primary_cell.py:11]: this is the same function repeated 3 times, we need to refactor (put it in one place) and call it in three different places.

**Review by gabdank — CHANGES_REQUESTED** [review]
needs to reduce code duplication

File: src/igvfd/tests/test_audit_primary_cell.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import pytest
+
+
+def test_audit_primary_cell_age(
+    testapp,
+    primary_cell
+):
+    res = testapp.get(primary_cell['@id'] + '@@index-data')
```
Comment (keenangraham) [inline src/igvfd/tests/test_audit_primary_cell.py:8]: I think should call `@@audit` here instead of `@@index-data`?

Comment (khzlin) [reply src/igvfd/tests/test_audit_primary_cell.py:8]: we've always used `@@index-data`, even in encoded. Should we be switching all to `@@audit`?

Comment (keenangraham) [reply src/igvfd/tests/test_audit_primary_cell.py:8]: Yes, I don't see a reason to use `@@index-data` here.

File: src/igvfd/audit/primary_cell.py (hunk 1 above)
Comment (khzlin) [reply src/igvfd/audit/primary_cell.py]: moving audit to biosample abstract level

Comment (khzlin) [reply src/igvfd/audit/primary_cell.py]: moving back to selective biosamples

**Review by gabdank — APPROVED** [review]

**Review by gabdank — APPROVED** [review]

**Pull request merged**